contributions/catalog/models/talent_utilities.py

```diff
@@ -36,42 +36,60 @@ def init_talent():
 def to_talent(d):
     if not d: return {}
     talent_list = []
-    if 'talent_name' in d.keys():
-        if isinstance(d['talent_name'], str):
+
+    # check how many capabilities are in the given json
+    # this should be checked keys that is as surfix of _number
+    num_tal = 0
+
+    # if there is talent_name_0, this means that there is talent
+    if 'talent_name_0' in d.keys():
+        keys = list(d.keys())
+        tal_len = []
+        # iterated talents to see how many talents are in there
+        for key in keys:
+            key_splitted = key.split("talent_name_")
+            if len(key_splitted) > 1:
+                tal_len.append(int(key_splitted[1]))
+        num_tal = max(tal_len) + 1
```

Comment:
can we utilize talent_count from contribute.html for this?

Reply:
That is the thing that I mentioned in the meetings... The communication between jinja and jquery is really hard.